### ISSUE TITLE
Is it possible "Get Text" in specific language using a different tesseract data?

### ISSUE THREAD

**Issue by romuluc** [rainmanwy/robotframework-SikuliLibrary#88]
Could I "Get Text" in portuguese ou spanish from OCR? Is there any way to calibrate OCR  to improve the quality of obtained text?

**Comment by edsonharantes** [comment]
Yes, you can, [take a look at this](https://sikulix-2014.readthedocs.io/en/latest/textandocr.html#textandocr), you can add a _traineddata_ data with your language and get better results, I'm from Brazil and I did this with portuguese, just look at the docs, don't forget to also configure the test cases to run with this configuration, I think there's a keyword for this.



👍

**Comment by rainmanwy** [comment]
Thank you @edsonharantes , @romuluc , "Get Text" is using tesseract, so need to add related trainedata.

**Comment by YiQiSun-Uniques** [comment]
Hi @rainmanwy, I download chi_sim.traineddata and copy it to folder tessdata where eng.traineddata exists in SikuliLibrary.jar, but it still cann't identify Chinese when I use RobotFramework, Do I need to modify other files?

Thanks in advanced

**Comment by rainmanwy** [comment]
@YiQiSun-Uniques , currently i could not have a try ocr in my environment.
Do you check the link as @edsonharantes suggested?

You may find sikulix folder in your home folder.
